src/server.rs

```diff
@@ -11,6 +12,14 @@ fn index(state: State<Database>) -> Json<Vec<Member>> {
     Json(state.get_members().clone())
 }
 
+#[get("/<id>")]
+fn show(state: State<Database>, id: String) -> Result<Json<Member>, NotFound<()>> {
```

Comment:
```suggestion
fn find_by_id(state: State<Database>, id: String) -> Result<Json<Member>, NotFound<()>> {
```

などの方が直感的でいいんじゃないかと思います

Reply:
なんか HTTP 層とリポジトリ層で分けたくて……
https://developer.twitter.com/en/docs/twitter-api/v1/tweets/post-and-engage/api-reference/get-statuses-show-id

Reply:
あ、なるほど。理解しました。